### ISSUE TITLE
Can't edit text in 2.1.3 and 2.1.4

### ISSUE THREAD

**Issue by iarczi** [rengwuxian/MaterialEditText#297]
after updating library to version 2.1.4 I can't edit text in my app. MaterialEditText view don't react on click - no keyboard, no cursor, nothing. Same in version 2.1.3, but in version 2.0.3 everything works alright.

**Comment by saantiaguilera** [comment]
I have this same issue.

**Comment by Hitexroid** [comment]
I have this same issue too.

**Comment by Hitexroid** [comment]
for solve this problem :+1: 

Add these style into your values -> style :

`<item name="windowNoTitle">true</item> <item name="windowActionBar">false</item> <item name="android:windowFullscreen">true</item> <item name="android:windowContentOverlay">@null</item>`

Add these style into your Activity tag or application tag:

`android:theme="@style/Theme.AppCompat.Light.NoActionBar.FullScreen"
"`



for Basic4Android Users :+1: 

Add This Code to Manifest

`SetApplicationAttribute(android:theme, "@style/MyAppTheme")

CreateResource(values,theme.xml,
<resources>
<style name="MyAppTheme" parent="@style/Theme.AppCompat.Light.NoActionBar">
<item name="colorPrimary">#FF9800</item>
<item name="colorPrimaryDark">#F57C00</item>
<item name="colorAccent">#FFA726</item>
<item name="windowNoTitle">true</item>
<item name="windowActionBar">false</item>
<item name="android:windowContentOverlay">@null</item>
</style>
</resources>
)`

**Comment by saantiaguilera** [comment]
Yep, I did a workaround with the xml styles too when I had it. Sorry I didnt updated the issue with my solution before ! :(

**Comment by IDFDeveloper** [comment]
I am facing the same issue and after spending more than 4 hours and I solved the issue by adding this property in the `AndroidManifest.xml` file. I know it is different but it is working for me in the Samsung device.

> android:hardwareAccelerated="true"
